l10n_jp_summary_invoice_sale_partner/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module displays the sale partner in the summary invoice report.
```

Comment:
Let's be clear which parts of the 6 patterns we discussed are covered. The uncovered patterns should go into the ROADMAP.